build/pmm-client-docker/Dockerfile

```diff
@@ -1,20 +1,25 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi7/ubi
 RUN yum -y install iproute nc \
     && rm -rf /var/cache/yum
 
 COPY pmm2-client.tar.gz /tmp/pmm2-client.tar.gz
+RUN groupadd -g 1002 pmm-agent
```

Comment:
Any particular reason we use those IDs? If yes, please add a comment.

Reply:
yes. openshift

Reply:
Let's add this sacred knowledge to the comment 😆

Reply:
done